server/src/http_server/pages/blog/md/html.rs

```diff
@@ -33,9 +35,11 @@ impl Default for SyntaxHighlightingContext {
                 .get("base16-ocean.dark")
                 .expect("This theme exists in the defaults")
                 .clone(),
+            current_article_path: None,
```

Comment:
Ahh I don't _love_ having this `None` here...

I wonder if we want to make a new struct that we pass down to the Markdown stuff?

Like we can attach `SyntaxHighlightingContext` to State like we are here without this `current_article_path`

And then we could have a new struct maybe like `MarkdownRenderContext` that has two items. The `SyntaxHighlightingContext` and the `current_article_path`

That way we don't have to create this struct with a None and fill it in later.
Not a _big_ difference but I think it will make the purpose of each struct a bit better!

Reply:
fixed!